h/test/tweens_test.py

```diff
@@ -0,0 +1,57 @@
+# -*- coding: utf-8 -*-
+
+from pyramid.testing import DummyRequest as _DummyRequest
+from h import tweens
+
+
+class DummyRegistry():
```

Comment:
`pyramid.testing.DummyRequest` should already have a registry, so you probably don't need this.